adobe_vipm/flows/sync.py

```diff
@@ -58,10 +58,15 @@ def _add_missing_subscriptions(
     agreement: dict,
     subscriptions_for_update: set[str],
 ) -> None:
-    logger.info(f"Checking missing subscriptions for {agreement["id"]=}")
-    adobe_subscriptions = exclude_subscriptions_with_deployment_id(
-        adobe_client.get_subscriptions(agreement["authorization"]["id"], customer["customerId"])
-    )["items"]
+    deployment_id = get_deployment_id(agreement) or ""
+    logger.info(f"Checking missing subscriptions for {agreement["id"]=}, {deployment_id=}")
```

Comment:
In order to make my life easier with new rules, could you stop using f-strings in log statements